news/2019-11-19-osu-idol-2019-finals-community-voting.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "osu!idol 2019 Finals - Community Voting!"
+date: 2019-11-19 12:00:00 +0000
+---
+
+It has been another amazing year for our osu!idol competition! There are three finalists we have for you to enjoy - [aceredshirt13](https://osu.ppy.sh/users/13736238), [moof](https://osu.ppy.sh/users/8457202), and [iamALieee](https://osu.ppy.sh/users/10093980) - go give them a listen and vote for your favourite!
+
+![](/wiki/shared/news/2019-11-19-osu-idol-2019-finals-community-voting/banner.jpg)
+
+### What is osu!idol?
```

Comment:
don't need this header

Reply:
removed, was going off header formatting from last year